packages/core/upload/admin/.eslintrc

```diff
@@ -7,6 +7,28 @@
       "rules": {
         "import/extensions": "off"
       }
+    },
+    {
+      "files": ["**/*.ts", "**/*.tsx"],
+      "extends": ["custom/front/typescript"]
+    },
+    {
+      "files": ["./tests/*", "**/*.test.*"],
+      "env": {
+        "jest": true
+      },
+      "rules": {
+        /**
+         * So we can do `import { render } from '@tests/utils'`
+         */
+        "import/no-unresolved": "off",
```

Comment:
is this import not resolved correctly?

Reply:
We need this rule to avoid these errors `Unable to resolve path to module '@tests/utils'` in all the unit test files, we are using the same rule (with the same comment) in the .eslintrc file in the admin, content-manager and review-workflow